apps/style/applab/style.scss

```diff
@@ -1,31 +1,84 @@
 $root: '/blockly/media/applab/'; //TODO: Parameterize for asset pipeline
 
 div#visualization {
-  height: 600px;
+  height: 400px;
+  width: 400px;
 }
 
 /* Shrink the visualization area on small displays. */
+
+@media screen and (min-width: 1151px) {
+  div#visualization.responsive {
```

Comment:
Please note in comments that the @media rules in this file are modified programmatically from the javascript and should not be modified here. I think that should be enough to explain why the rules are duplicated here and in common.scss.